tests/run_test.sh

```diff
@@ -32,7 +32,7 @@ if [ "$1" = "--help" ]; then usage; exit; fi
 # Allow overriding the 'metamath' command using the METAMATH env variable
 cmd="${METAMATH:-metamath}"
 
-# Alternatively, use the '-c metamath' argument which overrides the env variable
+# Alternately, use the '-c metamath' argument which overrides the env variable
```

Comment:
```suggestion
# Alternatively, use the '-c metamath' argument which overrides the env variable
```
I believe the original was correct. "Alternatively" is used to give alternatives in the sense "you can do this, or alternatively you can do that", while "Alternately" is used for something that goes back and forth between values, as in "the lines are alternately long and short".

Reply:
You're right, I'll revert the change.